Gemfile.lock

```diff
@@ -57,17 +57,17 @@ GEM
     crack (0.4.2)
       safe_yaml (~> 1.0.0)
     diff-lcs (1.2.5)
-    domain_name (0.5.25)
+    domain_name (0.5.20160826)
```

Comment:
Amazing performance by this Gem developer. 20,160,801 patch releases! They really have been busy.